ml-agents-envs/mlagents_envs/logging_util.py

```diff
@@ -44,3 +49,14 @@ def set_log_level(log_level: int) -> None:
 
     for logger in _loggers:
         logger.setLevel(log_level)
+
+    if log_level == DEBUG:
+        formatter = logging.Formatter(fmt=DEBUG_LOG_FORMAT, datefmt=DATE_FORMAT)
+        for logger in _loggers:
```

Comment:
Would this for loop be a good candidate for a separate method `def set_all_formatters`?